UPGRADE.md

```diff
@@ -1,5 +1,180 @@
 # Upgrade Guide
 
+## Upgrading To 13.0 From 12.x
+
+### Minimum Versions
+
+The following required dependency versions have been updated:
+
+- The minimum PHP version is now v7.3
+- The minimum Laravel version is now v8.0
+- The minimum Stripe SDK version is now v7.39
+
+### Stripe API Version
+
+PR: https://github.com/laravel/cashier-stripe/pull/905
+
+The Stripe API version for Cashier 13.x will be `2020-08-27`. Even though Cashier uses this version, it's recommended that you upgrade your own settings [in your Stripe dashboard](https://dashboard.stripe.com/developers) to this API version as well after deploying the Cashier upgrade. If you use the Stripe SDK directly, make sure to properly test your integration after updating.
+
+#### Tax Percentage Removal
+
+Because of the upgrade to the new Stripe API version, the `taxPercentage`, `syncTaxPercentage` & `getTaxPercentageForPayload` methods were removed from Cashier. It's recommended that you upgrade to Stripe's new Tax Rates API. You can familiarize yourself with Stripe's guides on Tax Rates:
+
+Stripe migration guide: https://stripe.com/docs/billing/migration/taxes  
+Tax Rates documentation: https://stripe.com/docs/billing/taxes/tax-rates  
+Tax Rates on invoices: https://stripe.com/docs/billing/invoices/tax-rates  
+
+Using Tax Rates in Cashier is also documented here: https://laravel.com/docs/billing#subscription-taxes
+
+### Renames To Prices
+
+PR: https://github.com/laravel/cashier-stripe/pull/1166
+
+Cashier v13 comes with some substantial API breakages. To accommodate to Stripe's phasing out of the Plans API we've made the choice to complete migrate the public api to the new Prices terminology. Doing so will bring us on par with Stripe documentation and will ease the translation between Stripe docs and Cashier documentation & code.
+
+While the changes are numerous and too much to name, you can generally do the changes as a direct change of methods names like so:
```

Comment:
I think we will need to list each method that was changed. Otherwise the user will be left pretty blind.

Reply:
@taylorotwell done